src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -1857,7 +1857,7 @@ async function getPODInfoAndSendToPortal(operation: Operation, event: string = "
     const projectInfo = operation.projectInfo;
     const projectLocation = projectInfo.location;
     const projectID = projectInfo.projectID;
-    const projectName = projectInfo.projectName;
+    const projectName = projectLocation.split("/").pop();
```

Comment:
Do we know why projectInfo.projectName was undefined here? Wondering whether projectInfo was not updated somewhere before this step.

Reply:
Not sure why, to be honest.

Reply:
As discussed in the meeting we'll go forward with this change since projectName is mainly used as an identifier for users that do a `kubectl get po` etc. but for Codewind it makes little difference since labels are used for querying resources.